samson-docker-ansible/Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN curl -o /get-pip.py https://bootstrap.pypa.io/get-pip.py && \
 	rm /get-pip.py
 
 RUN pip install jmespath==0.9.3
+RUN pip install docker==3.5.0
```

Comment:
merge it together with line 21 in order to avoid creating another layer

https://www.codacy.com/blog/five-ways-to-slim-your-docker-images/